content/concepts/payment-types/partial-payments.md

```diff
@@ -8,9 +8,11 @@ labels:
 ---
 # Partial Payments
 
-In the default case, the `Amount` field of a [Payment transaction][] in the XRP Ledger specifies the exact amount to deliver, after charging for exchange rates and [transfer fees](transfer-fees.html). The "Partial Payment" flag ([`tfPartialPayment`](payment.html#payment-flags)) allows a payment to succeed by reducing the amount received instead of increasing the amount sent. Partial payments are useful for [returning payments](become-an-xrp-ledger-gateway.html#bouncing-payments) without incurring additional costs to oneself.
+The sender of any [Payment transaction][] can enable the ["Partial Payment" flag](payment.html#payment-flags) and send a payment which delivers less than the `Amount` field indicates. When processing any Payment, use the `delivered_amount` metadata field, not the `Amount` field. The `delivered_amount` is the amount a payment actually delivered.
 
-The amount of XRP used for the [transaction cost](transaction-cost.html) is always deducted from the sender’s account, regardless of the type of transaction.
+If a Payment does not enable the Partial Payment flag, the `Amount` field of a [Payment transaction][] in the XRP Ledger specifies the amount to deliver after charging for exchange rates and [transfer fees](transfer-fees.html). The Partial Payment flag ([`tfPartialPayment`](payment.html#payment-flags)) allows a payment to succeed by reducing the amount received instead of increasing the amount sent. Partial payments are useful for [returning payments](become-an-xrp-ledger-gateway.html#bouncing-payments) without incurring additional costs to oneself.
```

Comment:
```suggestion
By default, the `Amount` field specifies the amount to deliver after exchange and transfer fees. Partial payments, however, treat the `Amount` field as a max amount to send, reducing the amount received by the fees incurred. This behavior makes partial payments useful for [returning payments](become-an-xrp-ledger-gateway.html#bouncing-payments) without incurring additional costs to oneself.
```

Reply:
I'd like to bring back the word "exact" which was a pretty key word in the original phrasing.

Also, @oeggert, that suggestion isn't quite correct. Even with the Partial Payments flag enabled, the `Amount` field is not the "max amount to send". It's the "max amount to deliver". The `SendMax` field is (always) the "max amount to send". 

There are various complex things that can happen in cross-currency payments or with trust lines that have non-default quality settings, so it's just a little bit over-simplified to say that partial payments reduce the amount received by the fees incurred.

Reply:
@mDuo13 if the phrasing proposed in this PR isn't acceptable, can you suggest an alternative?